### ISSUE TITLE
custom template without handlebars not working as expected

### ISSUE THREAD

**Issue by unclescooby** [corejavascript/typeahead.js#57]
I have the following template code running fine with v0.10.5:

```
suggestion: function(data) {
  return '<p><div class="tt-name">' + data.name + '</div><span>review</span></p>'; 
}
```

In 0.10.5 the whole template is rendered inside the suggestion element. In v0.11.1 this is not rendering at all. If I remove the p tags it renders, but the outer template element and attributes (eg the tt-name class) are added to the parent tt-suggestion div.

Solution is to wrap the template in an extra div.

Am I missing something with my setup? Is this behavior by design or is this a bug?

Demo:  https://jsfiddle.net/b0dnxxfL/

**Comment by unclescooby** [comment]
Actually, the whole outer element is used for the parent. Replace the outer div with a span, and the parent becomes a span.

**Comment by jlbooker** [comment]
Seems to have been resolved in #61. If that's not the case, let me know and we'll reopen.
